src/components/AirdropWalletBalanceCard.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+
+import { CONST, AirdropWalletDetails, AirdropGoal } from 'app/consts';
+import { typography, palette } from 'app/styles';
+
+import { AirdropWalletBalance } from './AirdropWalletBalance';
+
+const i18n = require('../../loc');
+
+interface Props {
+  walletDetails: AirdropWalletDetails;
+}
+
+export const AirdropWalletBalanceCard: FC<Props> = ({ walletDetails }) => {
+  const airdropGoals: AirdropGoal[] = [
+    { threshold: 5, name: i18n.airdrop.walletsCarousel.shrimp },
+    { threshold: 25, name: i18n.airdrop.walletsCarousel.crab },
+    { threshold: 100, name: i18n.airdrop.walletsCarousel.shark },
+    { threshold: 1000, name: i18n.airdrop.walletsCarousel.whale },
+  ];
+
+  const readableOrder = [i18n.order.first, i18n.order.second, i18n.order.third, i18n.order.fourth];
+  const unreachedGoals = airdropGoals.filter((goal: AirdropGoal) => goal.threshold > walletDetails.balance);
+  // TODO: edge case of "all goals reached" not covered by designs. Awaiting UX input. For now returning last one - "whale"
+  const nextGoal = unreachedGoals[0] || airdropGoals[airdropGoals.length - 1];
+  const nextGoalIndex = airdropGoals.findIndex((goal: AirdropGoal) => goal.threshold === nextGoal.threshold);
+
+  return (
+    <View style={styles.walletCard}>
+      <AirdropWalletBalance
+        balance={walletDetails.balance}
+        walletName={walletDetails.name}
+        threshold={nextGoal.threshold}
+        footer={
+          <>
+            <Text style={styles.firstLine}>
+              {i18n.formatString(i18n.airdrop.walletsCarousel.nextGoal, {
```

Comment:
this text is different after and before the airdrop

Reply:
done